### PR TITLE
go mod packages updates in the driver and operator

### PR DESCRIPTION
**Pull request by badri-pathak**
## Pull request checklist

```release-notes

```

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature Enhancement
- [ ] Test Automation 
- [x] Code Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Community Operator listing 
- [x] Other (please describe):  Go packages have been updated to latest.

## What is the current behavior?

-

## What is the new behavior?

- There are changes in the controller  and manager functions of the operator. Need to be validation

## How risky is this change?
- [ ] Small, isolated change
- [ ] Medium, requires regression testing
- [x] Large, requires functional and regression testing

### CODE REVIEW

**Comment by badri-pathak** [comment]
Image: quay.io/badri_pathak/ibm-spectrum-scale-csi-driver:gomodpackagetest_v1

**Comment by Jainbrt** [comment]
HI @badri-pathak can you please do the same for operator too in the same PR ?

**Comment by badri-pathak** [comment]
Images:
quay.io/badri_pathak/ibm-spectrum-scale-csi-operator:gomodpackagetest_v4
quay.io/badri_pathak/ibm-spectrum-scale-csi-driver:gomodpackagetest_v4

**Comment by hemalathagajendran** [comment]
@badri-pathak Please fix the travis build errors

**Comment by badri-pathak** [comment]
Images:
quay.io/badri_pathak/ibm-spectrum-scale-csi-operator:gomodpackagetest_v4
quay.io/badri_pathak/ibm-spectrum-scale-csi-driver:gomodpackagetest_v5

**Review by hemalathagajendran — CHANGES_REQUESTED** [review]

File: driver/go.mod [hunk 1]
```diff
@@ -1,37 +1,39 @@
 module github.com/IBM/ibm-spectrum-scale-csi/driver
 
-go 1.22
+go 1.22.0
```
[the remark this answers is not in the document]
Comment (badri-pathak) [reply driver/go.mod]: updated the version

**Review by hemalathagajendran — APPROVED** [review]

**Review by deeghuge — APPROVED** [review]

**Pull request merged**